common-lib/src/main/scala/com/gu/mediaservice/model/Collection.scala

```diff
@@ -32,6 +36,14 @@ object Collection {
       Action("remove", uri, "DELETE")
     ))
   }
+
+  // We use this to ensure we are creating valid `Collection`s
+  def create(path: List[String], actionData: ActionData) = {
```

Comment:
we use `build` elsewhere for these types of helper functions - not sure if there's a particular convention to follow?